dummy_user_1:
  ros__parameters:
    timer_period: 0.25
    stability_topic_name: '/user_stability'
    trans_speed:  0.5
    user_id: 'user1'
    user_tinetti: 1.0
    x_pose: [-1.66, -1.89, -3.22, -2.53,  1.56,  3.83,  2.41,  2.22, -0.77,  0.02, -2.51, -3.38, -1.09 ]
    y_pose: [ 3.19,  5.28,  5.89, 15.08, 14.76, 13.59, 10.78,  7.82,  8.32, 14.99, 15.18,  5.87,  3.44 ]
    s_pose: [ 1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0,   1.0 ]
dummy_user_2:
  ros__parameters:
    timer_period: 0.25
    stability_topic_name: '/user_stability'
    trans_speed:  0.5
    user_id: 'user2'
    user_tinetti: 0.6
    x_pose: [-1.66, -1.89, -3.22, -3.97, -2.53,  1.56,  3.83,  2.41,  2.22, -0.77,  0.02, -2.51, -3.38, -1.09 ]
    y_pose: [ 3.19,  5.28,  5.89,  7.32, 15.08, 14.76, 13.59, 10.78,  7.82,  8.32, 14.99, 15.18,  5.87,  3.44 ]
    s_pose: [ 0.2,   0.2,   0.2,   0.2,   0.2,   0.2,   0.2,   0.2,   0.2,   0.2,   0.2,   0.2,   0.2,   0.2 ]
dummy_user_3:
  ros__parameters:
    timer_period: 0.25
    stability_topic_name: '/user_stability'
    trans_speed:  0.65
    user_id: 'user3'
    user_tinetti: 0.4
    x_pose: [ 2.01, -1.66, -1.89, -3.22, -3.97, -2.53,  1.56,  3.83,  2.41,  2.22, -0.77,  0.02, -2.51, -3.38, -1.09 ]
    y_pose: [ 3.06,  3.19,  5.28,  5.89,  7.32, 15.08, 14.76, 13.59, 10.78,  7.82,  8.32, 14.99, 15.18,  5.87,  3.44 ]
    s_pose: [ 0.4,   0.4,   0.4,   0.4,   0.4,   0.4,   0.4,   0.4,   0.4,   0.4,   0.4,   0.4,   0.4,   0.4,   0.4 ]
dummy_user_4:
  ros__parameters:
    timer_period: 0.25
    stability_topic_name: '/user_stability'
    trans_speed:  0.85
    user_id: 'user4'
    user_tinetti: 0.8
    x_pose: [ 2.01, -1.66, -1.89, -3.22, -3.97, -2.53,  1.56,  3.83,  2.41,  2.22, -0.77,  0.02, -2.51, -3.38, -1.09 ]
    y_pose: [ 3.06,  3.19,  5.28,  5.89,  7.32, 15.08, 14.76, 13.59, 10.78,  7.82,  8.32, 14.99, 15.18,  5.87,  3.44 ]
    s_pose: [ 0.56,  0.59,  0.58,  0.89,  0.32,  0.58,  0.76,  0.59,  0.78,  0.82,  0.32,  0.99,  0.01,  0.07,  0.24 ] 